chaos_genius/views/anomaly_data_view.py

```diff
@@ -47,6 +45,18 @@ def kpi_anomaly_detection(kpi_id):
     end_date = None
     try:
         kpi_info = get_kpi_data_from_id(kpi_id)
+
+        if not kpi_info["anomaly_params"]:
+            current_app.logger.info(f"No Anomaly Data Found for KPI ID: {kpi_id}")
```

Comment:
This log message here could be better indicative of the error.
```suggestion
            current_app.logger.info(f"Could not get anomaly data since anomaly is not configured for KPI ID: {kpi_id}")
```

Reply:
Also, why don't we return this message in the `msg` field?